Add a section about severity level to the guidelines (#90)
neo4j/docs-status-codes: README.adoc

| The label `Perso` does not exist.
|===

[[severity-level]]
==== Severity level

The severity can be one of the following:

* `WARNING`: The query may have issues. Please review it. We might be able to give you a suggestion on how to improve it.

> * `INFORMATION`: The query may have issues, even though it is correct. The provided information can still be helpful.

Reviewer: I would like to also add that we are generally more certain the query have issues for WARNING compar.ed to INFORMATION. Maybe @LinneaAndersson has some feedback too

Author: How about:
* `WARNING`: It is about queries/commands that may have structural, syntactic, semantic, or spelling issues that prevent them from being executed in an optimal way. We need to give example(s) and suggestion(s) for improvement.
* `INFORMATION`: It is about queries/commands that are correct but may have some performance issues or that have no effect because they try to do something already done. Depending on the case, we may give example(s) and suggestion(s) for improvement.

`Neo.ClientNotification.Database.HomeDatabaseNotFound` is the only status code that deviates from the above definition for `INFORMATION`.

Reviewer: I think that is better 👍

> * `ERROR`: The query execution failed. Please review the query. We might be able to give you a suggestion on how to improve it.

Author: ```suggestion
* `ERROR`: It is about queries/commands that fail to succeed. It's good to tell the user what triggers the error, an example, and a possible solution(s).
```

| The label `Perso` does not exist.
|===

[[severity-level]]
==== Severity level

The severity can be one of the following:

> * `WARNING`: It is about queries/commands that may have structural, syntactic, semantic, or spelling issues that prevent them from being executed in an optimal way.

Reviewer 2: I would more say that:
* `WARNING`: It is about queries/commands that may have structural, syntactic, semantic, or spelling issues where we believe the query most probably is wrong and/or should be updated. This includes: deprecations, experimental features, trying to match on non-existing labels etc. It should be clear what the problem is and it is also good to tell the possible cause, and the consequences, as well as give an example(s) and suggestion(s) for improvement.

Reviewer 2: I think the most important part is that "we are (almost) certain it is a problem" and we know how to fix your query. So for:

- Deprecations -> you need to update the query with new functionallity
- Experimental features -> These can not be trusted and should be updated if  used in production. 
- Non-existing label: This query will never give any result, so it's either useless to run OR misspelled label (unless you add a node with the label, but then the notification would disappear).

Author: I like that, and I will add it as it is.

| The label `Perso` does not exist.
|===

[[severity-level]]
==== Severity level

The severity can be one of the following:

* `WARNING`: It is about queries/commands that may have structural, syntactic, semantic, or spelling issues where the query most probably is wrong and/or should be updated.
This includes deprecations, experimental features, trying to match on entities with non-existing labels, etc.
It should be clear what the problem is and it is also good to tell the possible cause, and the consequences, as well as give an example(s) and suggestion(s) for improvement.

> I think the most important part is that "we are (almost) certain it is a problem" and we know how to fix your query. So for:

Reviewer 2: ```suggestion
The most important part is that "we are (almost) certain it is a problem" and we know how to fix your query. For example:
```

Author: Someone was distracted 😄 and forgot to amend it afterward.
